pkg/cmd/generate.go

```diff
@@ -76,3 +79,27 @@ func Generate(paths []string) error {
 	}
 	return nil
 }
+
+func validateOutput(data []byte) {
+	results, err := kubeval.Validate(data, "output")
+	if err != nil {
+		errors.Wrap(err, "Unable to validate")
+		os.Exit(1)
+	}
+	success := true
+	for _, result := range results {
+		if len(result.Errors) > 0 {
+			success = false
+			fmt.Println("The document", result.FileName, "contains an invalid", result.Kind)
+			for _, desc := range result.Errors {
```

Comment:
we don't even need to iterate over all the errors, just club them to form a single error and return it.